pkg/service/sniffer/runtime/docker.go

```diff
@@ -25,11 +34,11 @@ func (d DockerBridge) ExtractPid(inspection string) (*string, error) {
 	panic("Docker doesn't need this implemented")
 }
 
-func (d DockerBridge) BuildTcpdumpCommand(containerId *string, netInterface string, filter string, pid *string) []string {
```

Comment:
using a pointer here should fix the cleanup command as well, since now `tcpdumpContainerName` should be persisted on the struct